queue_services/events-listener/requirements/repo-libraries.txt

```diff
@@ -1,3 +1,4 @@
 -e git+https://github.com/bcgov/lear.git#egg=entity_queue_common&subdirectory=queue_services/common
 -e git+https://github.com/bcgov/sbc-common-components.git#egg=sbc-common-components&subdirectory=python
--e git+https://github.com/bcgov/sbc-auth.git@development#egg=auth-api&subdirectory=auth-api
+# -e git+https://github.com/bcgov/sbc-auth.git@development#egg=auth-api&subdirectory=auth-api
```

Comment:
same as above. should it be changed back to bcgov?

Reply:
same comment as above..I will do another PR right after this to fix this...is that acceptable?

Reply:
Sounds good.